python/lsst/afw/math/chebyshevBoundedField.cc

```diff
@@ -21,16 +21,72 @@
  */
 
 #include <pybind11/pybind11.h>
-//#include <pybind11/operators.h>
-//#include <pybind11/stl.h>
+#include <pybind11/operators.h>
+#include <pybind11/stl.h>
+
+#include "numpy/arrayobject.h"
+#include "ndarray/pybind11.h"
+#include "ndarray/converter.h"
+
+#include "lsst/afw/math/BoundedField.h"
+#include "lsst/afw/math/ChebyshevBoundedField.h"
 
 namespace py = pybind11;
+PYBIND11_DECLARE_HOLDER_TYPE(MyType, std::shared_ptr<MyType>);
 
 using namespace lsst::afw::math;
 
 PYBIND11_PLUGIN(_chebyshevBoundedField) {
     py::module mod("_chebyshevBoundedField", "Python wrapper for afw _chebyshevBoundedField library");
+    
+    if (_import_array() < 0) {
+            PyErr_SetString(PyExc_ImportError, "numpy.core.multiarray failed to import");
+            return nullptr;
+    };
```

Comment:
We probably want to make a convenience function for these kind of things in the future. Don't worry about it now, but I'll add something soon.